lib/Bio/EnsEMBL/DataCheck/Checks/CheckHomologyMLSS.pm

```diff
@@ -43,6 +43,9 @@ sub tests {
   my $dba    = $self->dba;
   my $helper = $dba->dbc->sql_helper;
   my @method_links = qw(ENSEMBL_ORTHOLOGUES ENSEMBL_PARALOGUES ENSEMBL_HOMOEOLOGUES ENSEMBL_PROJECTIONS);
+  if ($dba->dbc->dbname !~ /[ensembl_compara|protein_trees|ncrna_trees]/) {
```

Comment:
```suggestion
  if ($dba->dbc->dbname ~= /[homology_annotation]/) {
```
Just in case our production pipelines change name in the future (e.g. with the new gene tree pipeline).

Reply:
Then in the future we will change the DC to reflect new gene tree pipeline name - the per-species compara databases will not be called "homology_annotation".